data/mods/Magiclysm/monsters/domestic.json

```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "type": "MONSTER",
+    "id": "mon_giant_heron_ridable",
+    "name": { "//~": "Word is invented: meaning is given as 'white-striders' ", "str": "aenor", "str_pl": "aenóren" },
+    "description": "Graceful and resembling herons but stretched to the size of a horse, the aenóren (\"white-striders\" in Brocéliandean) were domesticated by the elves millennia ago.  While not useful as draft beasts, they retained popularity as courier or racing mounts due to their speed, at least until the invention of automobiles.",
+    "species": "BIRD",
+    "default_faction": "fantasy_beast_domestic",
+    "bodytype": "ostrich",
+    "symbol": "H",
+    "color": "blue_white",
+    "volume": "550 L",
+    "weight": "500 kg",
+    "material": "flesh",
+    "aggression": 0,
+    "aggro_character": false,
+    "hp": 250,
+    "morale": 50,
+    "speed": 350,
+    "attack_cost": 390,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 8,
+    "melee_damage": [ { "damage_type": "cut", "amount": 6 } ],
+    "grab_strength": 5,
+    "dodge": 3,
+    "mountable_weight_ratio": 0.25,
+    "special_attacks": [
+      { "type": "leap", "cooldown": 5, "max_range": 5, "allow_no_target": true, "move_cost": 350 },
+      { "id": "impale", "cooldown": 10, "move_cost": 375 },
+      { "id": "mutant_hoof", "dodgeable": true, "move_cost": 350 }
+    ],
+    "reproduction": { "baby_type": { "baby_egg": "egg_giant_heron_ridable" }, "baby_count": 1, "baby_timer": 540 },
+    "baby_flags": [ "SPRING" ],
+    "flags": [
+      "SEES",
+      "SMELLS",
+      "HEARS",
+      "GOODHEARING",
+      "ANIMAL",
+      "PATH_AVOID_DANGER",
+      "WARM",
+      "HIT_AND_RUN",
+      "PET_MOUNTABLE",
+      "PET_WONT_FOLLOW",
+      "CANPLAY",
+      "GROUP_MORALE"
+    ],
+    "biosignature": { "biosig_item": "feces_bird", "biosig_timer": 3 },
+    "harvest": "large_bird_flightless",
+    "dissect": "dissect_bird_sample_large",
+    "zombify_into": "mon_giant_heron_ridable_zombie",
+    "anger_triggers": [ "FRIEND_ATTACKED", "PLAYER_NEAR_BABY" ],
+    "fear_triggers": [ "FIRE", "FRIEND_DIED", "HURT", "PLAYER_CLOSE" ],
+    "petfood": {
+      "food": [ "BIRDFOOD" ],
```

Comment:
Since birdfood is sized for small birds would it make sense to add a larger version for these birds?

Reply:
I thought about that but when I went to check on the birdfood item it has 400(!) calories, way more than cattle fodder's 170 even though it's a much smaller volume, so I thought a separate item wasn't necessary.

Reply:
Absolutely incredible.